docs/developers/shuffleclient.md

```diff
@@ -124,19 +124,25 @@ to guarantee no data is lost.
 ```java
   public abstract CelebornInputStream readPartition(
       int shuffleId,
+      int appShuffleId,
       int partitionId,
       int attemptNumber,
       int startMapIndex,
-      int endMapIndex)
+      int endMapIndex,
+      ExceptionMaker exceptionMaker,
+      MetricsCallback metricsCallback)
 ```
 
-- `shuffleId` is the unique shuffle id of the application
+- `shuffleId` is the unique shuffle id of Celeborn
```

Comment:
This description is not correct. This can be original shuffle id or remapped shuffle id.
You can refer this PR #1924.